src/DefaultCronExpressionExplainer.php

```diff
@@ -62,6 +62,7 @@ public function getSupportedLocales(): array
 	{
 		return [
 			'cs' => 'czech',
+			'sk' => 'slovak',
 			'en' => 'english',
 		];
```

Comment:
Please keep alphabetical order and also add the new locale to docs/README.md

```suggestion
		return [
			'cs' => 'czech',
			'en' => 'english',
			'sk' => 'slovak',
		];
```